airbyte-integrations/connectors/source-merge/source_merge/manifest.yaml

```diff
@@ -0,0 +1,76 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["results"]
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.merge.dev/api/ats/v1"
+    http_method: "GET"
+    request_headers:
+      X-Account-Token: "{{ config['account_token'] }}"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_token'] }}"
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "#/definitions/requester"
+
+  base_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/retriever"
+  
+  account_details_stream:
+    type: DeclarativeStream
+    retriever:
+      type: SimpleRetriever
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: []
+      paginator:
+        type: NoPagination
+      requester:
+        $ref: "#/definitions/requester"
+    name: "account_details"
+    primary_key: "id"
+    $parameters:
+      path: "/account-details"
+
+  sync_status_stream:
+    $ref: "#/definitions/base_stream"
+    name: "sync_status"
+    primary_key: "model_id"
+    $parameters:
+      path: "/sync-status"
+
+  users_stream:
+    $ref: "#/definitions/base_stream"
+    name: "users"
+    primary_key: "id"
+    $parameters:
+      path: "/users"
```

Comment:
It is missing the incremental sync capability and pagination.

Reply:
Can you try to use a manual curl request? It doens't make sense to not work if the API docs says so.

Reply:
Tried with Postman, See remote_created_at field

![image](https://user-images.githubusercontent.com/47392334/233403555-e65386fb-e9b2-4952-81b1-14fa8bc79a50.png)

Reply:
You need to have more records inside the users to have pagination and the documentations is saying to use created_after

Reply:
Uh. Little problem with that. See https://github.com/airbytehq/airbyte/pull/25342#issuecomment-1516473041